pkg/config/config_test.go

```diff
@@ -121,6 +121,41 @@ var _ = Describe("Config", Ordered, func() {
 			Expect(myconf).To(BeNil())
 		})
 	})
+
+	When("Persisting tokens", Ordered, func() {
+		var tmpdir string
+		var err error
+
+		BeforeAll(func() {
+			tmpdir, err = os.MkdirTemp("/tmp", ".ocm-config-*")
+			Expect(err).To(BeNil())
```

Comment:
NP: we've been moving towards `Expect(err).ToNot(HaveOccurred())`

Reply:
updated, thanks